math/src/field/fields/u64_prime_field.rs

```diff
@@ -56,6 +56,24 @@ impl<const MODULO: u64> HasFieldOperations for U64PrimeField<MODULO> {
     }
 }
 
+impl<const MODULO: u64> IsLinearField for U64PrimeField<MODULO> {
+    fn bit_size() -> usize {
+        (Self::BaseType::BITS - (MODULO - 1).leading_zeros()) as usize
```

Comment:
The bit size is the minimum amount of bits to represent a number in the field.
The idea is simple: "total numbers of base type" minus the "total amount of leading zeros of the biggest element".

Honestly, I'd like this method to be a `const` method since this can be resolved at compile time, but we can't have `const` methods in traits. I'm open to ideas if it sound worth it.